nuget/runners/nunit.console-runner-with-extensions.nuspec

```diff
@@ -34,7 +34,7 @@
         <dependency id="NUnit.Extension.VSProjectLoader" version="$version$" />
         <dependency id="NUnit.Extension.NUnitV2ResultWriter" version="$version$" />
         <dependency id="NUnit.Extension.NUnitV2Driver" version="$version$" />
-        <dependency id="NUnit.Extension.TeamCityEventListener" version="1.0.1" />
+        <dependency id="NUnit.Extension.TeamCityEventListener" version="[1.0.*,1.1)" />
```

Comment:
As I understand - we are planning to have ability to do independent releases. If I defined `1.0.2` all users would install 1.0.1 according to rule `Lowest applicable version` When I set `[1.0.*,1.1)` all users are able to get hot-fixes in this range, according to rule `Floating Versions`, and we should not make a new package `nunit.runners` each time when we would like to publish hot-fix. Although maybe that something is misunderstood in http://docs.nuget.org/ndocs/consume-packages/dependency-resolution